frontends/mit-open/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "watch": "API_BASE_URL=http://localhost:8063 NODE_ENV=development ENVIRONMENT=local webpack serve",
     "watch:docker": "API_BASE_URL=http://nginx:8063 NODE_ENV=development ENVIRONMENT=local webpack serve",
-    "watch:rc": "API_BASE_URL=https://mitopen-rc.odl.mit.edu/ NODE_ENV=development ENVIRONMENT=local webpack serve",
+    "watch:rc": "API_BASE_URL=https://api.mitopen-rc.odl.mit.edu/ NODE_ENV=development ENVIRONMENT=local webpack serve",
```

Comment:
This change seems unrelated to this PR. Can we break it out into a separate one for tracking purposes, and in case it needs to be reverted?

Reply:
PR here: https://github.com/mitodl/mit-open/pull/1022